### ISSUE TITLE
peerDependencies Issue

### ISSUE THREAD

**Issue by 45akshat** [CleverProgrammers/LIVE-uber-eats-clone-youtube#5]
This repository was started long back and i think it is no longer been maintained and because of this when you install the modules it wont work as everything is supported and linked anymore. PLEASE revert if I am wrong...

**Comment by carlitoSwaaay** [comment]
> This repository was started long back and i think it is no longer been maintained and because of this when you install the modules it wont work as everything is supported and linked anymore. PLEASE revert if I am wrong...

I agree, redux is on version 9 now and was reworked in several ways. The code won't work as is. I'm working on updating it.
